server/public/scripts/client.js

```diff
@@ -48,3 +48,30 @@ function saveKoala( newKoala ){
   // ajax call to server to get koalas
  
 }
+
+function deleteKoala(){
+  $.ajax({
+    type: 'DELETE',
+    url: `/koala/${$(this).parents('tr').data('id')}`
```

Comment:
There should be a 'data-id=""' somewhere in the append function.  Also, this references 'tr' with no 'tr' in append.